index.html

```diff
@@ -1,432 +1,503 @@
-          startTime: 20
-        },
-        {
-          id: 'tEddixS-UoU',  // Janelle Monae QUEEN
-          startTime: 53
-        },
-        {
-          id: 'ClU3fctbGls', // P!nk What About Us
-          startTime: 40,
-          endTime: 300
-        },
-        {
-          id: 'k2qgadSvNyU',  // Dua Lipa New Rules
-          startTime: 10
-        },
-        {
-          id: 'eppa0RXRU-I'  // Nicki Minaj Barbie Tingz
-        },
-        {
-          id: 'kcRWyLMVrFM', // Cardi B Ring
-          startTime: 7
-        }
+  <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <title>CrossFadeYT | Custom Crossfading Music Video Playlists on YouTube</title>
+    <style>
+      body {
+        padding: 0;
+        margin: 0;
+        color: #000;
+        background-color: #FFFFFF;
+        font-family: Helvetica, Arial, sans-serif;
+      }
+
+      button {
+        background-color: white;
+        border: 0px;
+        border-radius: 15px;
+        padding: 5px;
+        font-family: Helvetica, Arial, sans-serif;
+        height: 30px;
+        width: 140px;
+        margin: 10px 10px;
+        border: 1px solid #ddd;
+        box-shadow: 1px 1px 1px #ddd;
+        text-transform: capitalize;
+      }
+
+      input {
+        border-radius: 5px;
+        padding: 10px;
+        display: block;
+        margin: auto;
+        width: 200px;
+        height: 50px;
+        text-align: center;
+      }
+
+      a{
+        text-decoration: none;
+        color: #fff;
+      }
+
+      .logo{
+        font-size: 40px;
+        background-color: #3D6C92;
+        color: #fff;
+        height: 70px;
+        text-align: center;
+        font-weight: bold;
+        line-height: 70px;
+      }
+
+      .top-nav{
+        background-color: #3D6C92;
+        color: #fff;
+        height: 70px;
+        font-weight: bold;
+        line-height: 70px;
+      }
+
+      .song-input {
+        padding: 10px;
+      }
+
+      .container {
+        padding: 0px 0px;
+      }
+
+      .footer {
+        margin: 0px 100px;
+        color: #000;
+        font-style: italic;
+        text-align: center;
+        padding: 10px;
+      }
+
+      .gh-img {
+        width: 50px;
+        display: block;
+        margin: 20px auto;
+      }
+
+      .left-side {
+        width: 30%;
+        min-height: 100vh;
+        background-color: #eee;
+      }
+
+      .right-side {
+        width: 70%;
+      }
+
+      .details{
+        color: #aaa;
+        text-align: center;
+      }
+
+      .add-song-button{
+        background-color: #fff;
+        border-bottom: 1px solid #ddd;
+        text-align: center;
+      }
+
+      .right-buttons{
+        border-bottom: 1px solid #ddd;
+      }
+
+      .selected{
+        background-color: #2C72C7;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="container">
+
+      <div class="main-page" style="display: flex;flex-direction: row">
```

Comment:
move these styles to the .main-page styles above

Reply:
ok